.circleci/config.yml

```diff
@@ -182,6 +180,11 @@ jobs:
           at: ~/
       # install gcloud orb
       - gcp-cli/install
+      - run:
+          name: Inject Heap Analytics Script to index.html
+          command: |
+            sed -i 's/HEAPIO_APP_ID/${HEAPIO_APP_ID}/' public/heapanalytics.js
+            sed -i 's|</head>| i <script src="/heapanalytics.js"></script>|' public/index.html
```

Comment:
The working directory of the job is `back/`, and we want to modify the build artifact, so the file path should be `../react/build/public/index.html` I think.

Reply:
Same for the JS script